src/engine/renderer/tr_image.cpp

```diff
@@ -1381,6 +1321,9 @@ image_t        *R_AllocImage( const char *name, bool linkIntoHashTable )
 		r_imageHashTable[ hash ] = image;
 	}
 
+	// Default image depth.
```

Comment:
I suggest not calling it numLayers either because layers have a specific meaning for array textures.

Reply:
Yes we already use “layer” for 3D textures, and images with such layers also have a color depth.

Reply:
Isn't that 6 for cubemaps? Since cubemaps are distinct from 3D textures, and there are, for example, cubemap arrays which have the amount of layers equal to the amount of cubemaps.

Reply:
Done.